packages/swift/Sources/Generated.swift

```diff
@@ -15,25 +15,53 @@ public protocol Event: Codable {}
 
 public enum WidgetEvent {
     public struct Load: Event {
+
+        public static func == (lhs: WidgetEvent.Load, rhs: WidgetEvent.Load) -> Bool {
```

Comment:
Just to make sure I follow:

- This is using pattern matching to match when `==` is used on two `WidgetEvent.Load` types? Which is why we can hardcode it to `true`

Reply:
No pattern matching, this particular struct doesn't have any members so there's nothing to compare. They're all equivalent to another.

Reply:
Honestly we should probably fix the load even on the widget side. I think it should have the user/session guid. But LGTM for now.